prusti-viper/src/encoder/foldunfold/footprint.rs

```diff
@@ -68,6 +68,26 @@ impl ExprFootprintGetter for vir::Expr {
                 &right.get_footprint(predicates),
             ),
 
+            vir::Expr::ContainerOp(_, box left, box right, _) => union(
+                &left.get_footprint(predicates),
+                &right.get_footprint(predicates),
+            ),
+
+            vir::Expr::Seq(_, elems, _) => {
+                let elem_perms = elems
+                    .iter()
+                    .map(|e| {
+                        e.get_footprint(predicates)
+                    })
+                    .collect::<Vec<_>>();
```

Comment:
I see there is [`FromIterator` on `HashSet`](https://doc.rust-lang.org/std/iter/trait.FromIterator.html#impl-FromIterator%3CT%3E-7) – does it work to just `.collect:<HashSet<_>>()` here?

Reply:
there is, but for `T`, where `e.get_footprint` returns a `Vec<T>`. i guess i could get rid of the intermediate `elem_perms` vec, but i haven't found a simple way to merge all `HashSet`s from the iter without the temporary variables. 
i guess i could use `fold`, what do you think abou
```
            vir::Expr::Seq(_, elems, _) => {
                elems
                    .iter()
                    .map(|e| {
                        e.get_footprint(predicates)
                    })
                    .fold(HashSet::new(), |hs, fp| {
                        hs.extend(fp); hs
                    })
            }
```

Reply:
I see, that's unfortunate. The current version works then.